test/test-resize-to-last-concurrent.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var cluster = require('cluster');
+var master = require('../lib/master');
+var tap = require('tap');
+var workerCount = require('./helpers').liveWorkers;
+
+cluster.setupMaster({
+  exec: 'test/workers/null.js'
+});
+
+tap.test('to last of concurrent resizes', function(t) {
+  master.start({size: 10});
+  master.on('startWorker', function() {
+    if (workerCount() === 3) {
+      master.setSize(5);
+    }
+    if (workerCount() === 5) {
```

Comment:
Is the lack of `else` because `master.setSize(5);` is expected to be synchronous and both of these conditions can actually be hit?